### PR TITLE
fix: allow admins to download all apps

### PR DESCRIPTION
**Pull request by erikarenhill**
this adds possibility to authenticate the user on the download endpoint for apps, and passes the JWT-token in the querystring on download links when viewing an unapproved/rejected app in the admin/edit-version of the gui.

This also brings in some changes to how a developer is determined to have access to an app. Previously it would only allow edit-access if being the developer that uploaded the app, now it will allow access to all apps which you belong to the same organisation it's connected to.

Also tested that you can retreive "myapps" with a user having multiple organisations

Adds transaction/rollback beforeEach/afterEach organisation unit tests

### CODE REVIEW

File: server/src/routes/v1/apps/handlers/getAppFile.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
+const Boom = require('@hapi/boom')
+
 const debug = require('debug')('apphub:server:routes:handlers:v1:getAppFile')
 const { AppStatus } = require('../../../../enums')
 const { getFile } = require('../../../../utils')
 
+const { currentUserIsManager } = require('../../../../security')
+
 module.exports = {
     //unauthenticated endpoint returning the approved app for the specified id
```
Comment (Birkbjo) [inline server/src/routes/v1/apps/handlers/getAppFile.js:10]: Comment is outdated

Comment (Birkbjo) [inline server/src/routes/v1/apps/handlers/getAppFile.js:10]: Comment is outdated

File: client/src/components/appVersion/VersionListEdit.jsx [hunk 2]
```diff
@@ -210,11 +210,18 @@ class VersionListEdit extends Component {
 
         const values = merge(version, this.state.editedValues[version.id])
 
+        //auth0 stores the JWT token in localStorage
+        //as only authenticated users can edit an app, just assume this exists in this component
+        const token = localStorage.getItem('id_token')
```
[the remark this answers is not in the document]
Comment (erikarenhill) [reply client/src/components/appVersion/VersionListEdit.jsx]: of course, forgot we had this!

File: server/src/routes/v1/apps/handlers/getAppFile.js (hunk 1 above)
Comment (erikarenhill) [reply server/src/routes/v1/apps/handlers/getAppFile.js]: removed

**Pull request merged**